composer.json

```diff
@@ -50,17 +50,17 @@
     "require":           {
         "php":                                  "^7.3 || ^8.0",
         "composer/package-versions-deprecated": "^1.11",
-        "doctrine/dbal":                        "^2.10",
+        "doctrine/dbal":                        "^2.12 || ^3.0",
```

Comment:
Is there a certain reason for 2.12 and 3.0? These versions seem to be unsupported, so maybe we should use ^2.13 and ^3.1?

Reply:
Doctrine documentation says 2.10.2 is current version (probably wrong). Anyway, if we can keep older version in low cost (like now), that we can do it, because why not?

Reply:
Could it be that you accidentaly looked at orm and not a dbal? For orm 2.10.2 is indeed current, but for dbal https://www.doctrine-project.org/projects/dbal.html says that 2.13.4 and 3.1.3 are both current and supported.